visearch-android/src/main/java/com/visenze/visearch/android/data/ProductTypeData.java

```diff
@@ -0,0 +1,60 @@
+package com.visenze.visearch.android.data;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.json.JSONObject;
+
+import java.util.List;
+import java.util.Map;
+
+
+public class ProductTypeData {
```

Comment:
why not combine with ProductType and add in the annotations ? you can always rename box to boxArray to prevent conflict, upon setBoxArray, set the box object.

Reply:
removed this already